### PR TITLE
A little clean up. Refactored config.js

### PR DESCRIPTION
**Pull request by marek-siemieniuk-morawski**
## What does this PR do?

I removed at least some of the unnecessary files and refactored `config.js`. I migrated it to Typescript and moved to `src/`. Also, as you noticed, I removed all values from the file - I simply believe our `.env` files should be the only source of truth and having configuration anywhere else might result in unexpected behaviors. The app should simply crash if any of the env variables are missing. This way it's more predictable.

tl;dr the most important changes are made it `src/config/index.ts`.

## Todos:

- [X] Checked on iOS
- [X] Checked on Android

### CODE REVIEW

**Comment by KedziaPawel** [comment]
would be nice to move  everything to src, like class/ models/ logger BlueApp.js BlueElectrum.js etc.

**Review by KedziaPawel — APPROVED** [review]

File: class/segwit-bech-wallet.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import config from '../config';
+import config from 'app/config';
```
Comment (KedziaPawel) [inline class/segwit-bech-wallet.js:1]: won't work

File: App.tsx [hunk 2]
```diff
@@ -5,13 +5,12 @@ import { View, StyleSheet, LogBox } from 'react-native';
 import { Provider } from 'react-redux';
 import { PersistGate } from 'redux-persist/integration/react';
 
+import config from 'app/config';
```
Comment (KedziaPawel) [inline App.tsx:8]: +1

File: class/watch-only-wallet.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import config from '../config';
+import config from 'app/config';
```
Comment (KedziaPawel) [inline class/watch-only-wallet.js:1]: +1

File: utils/bitcoin.ts [hunk 4]
```diff
@@ -1,7 +1,8 @@
 import * as bitcoin from 'bitcoinjs-lib';
 import { round } from 'lodash';
 
-import config from '../config';
+import config from 'app/config';
```
Comment (KedziaPawel) [inline utils/bitcoin.ts:4]: +1
